compiler/modules/nback/structure.bal

```diff
@@ -352,30 +352,48 @@ function mappingOrderFields(t:MappingAtomicType mat, string[] fieldNames, bir:Op
 function buildMappingGet(llvm:Builder builder, Scaffold scaffold, bir:MappingGetInsn insn) returns BuildError? {
     bir:Register mappingReg = insn.operands[0];
     bir:StringOperand keyOperand = insn.operands[1];
-    int? fieldIndex = mappingFieldIndex(scaffold.typeContext(), mappingReg.semType, keyOperand);
-    boolean fill;
-    RuntimeFunction rf;
-    llvm:Value k;
-    if fieldIndex == () {
-        fill = insn.name == bir:INSN_MAPPING_FILLING_GET;
-        rf = fill ? mappingFillingGetFunction : mappingGetFunction;
-        k = check buildString(builder, scaffold, keyOperand);
+    var fieldIndexRes = mappingFieldIndex(scaffold.typeContext(), mappingReg.semType, keyOperand);
```

Comment:
This is something like `compTimeMappingGetInfo` but that is ugly. Maybe `knowMappingField` ? Let's discuss in the meeting.